source/components/molecules/FileUploaderList/test/FileUploaderList.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { render } from "../../../../../test-utils";
+import FileUploaderList from "../FileUploaderList";
+
+describe("FileUploaderList", () => {
```

Comment:
I would like to see a test where we are testing the onChange method and that its called with correct parameters depending on which FilePicker is used in the list. I assume that the onChange callback will be called with different parameters depending on which FilePicker is used right?